docs/sources/helm-charts/tempo-distributed/get-started-helm-charts/_index.md

```diff
@@ -480,32 +480,32 @@ The following example configuration sets some global configuration options, as w
 
 ```yaml
 overrides:
-    defaults:
```

Comment:
This should be consistent with our other examples on the page which are 4 spaces, this aligned to 2. Conversely if our standard is 2 then the examples above should be 2 also.